quickstarts/mlops/truera/dashboards/truera.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "Truera: Initial Dashboard",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```

Comment:
If you could also remove the permissions parameter that would be great too. This parameter is in the json when you copy a dashboard, but it causes issues when you go to create a Dashboard using JSON
```suggestion
```

Reply:
removed description field based on comment in another PR and permission field based on comment here.